app/src/main/res/values/strings.xml

```diff
@@ -420,6 +420,7 @@
     <string name="lock_account_label">Lock account</string>
     <string name="lock_account_label_description">Requires you to manually approve followers</string>
     <string name="compose_save_draft">Save draft?</string>
+    <string name="compose_save_draft_loses_media">Save draft? (You will lose all attachments and any captions)</string>
```

Comment:
Nothing will be lost. It will just be re-uploaded. 
How about "Attachments will be uploaded again when you restore the draft"

Reply:
Hmm, appears that I'm just not yet comfortable w/ the UI... I found the draft whose picture+caption I thought I lost (everything intact). Now I wonder what happened that I ended up elsewhere ... Perhaps I ended up in a new compose window?

Anyway, your text sounds fine.